commons/src/main/java/org/infinispan/commons/util/concurrent/jdk8backported/ForkJoinTask.java

```diff
@@ -1,1512 +0,0 @@
-// Revision 1.12
```

Comment:
These 2 classes were removed as they are already in core.  They actually cause compilation issues if using an older JVM where certain classes like CountedCompleter are not available (we have it copied in core).